governance-exercises/Le Bateleur - Interpretation Exercise 01/GOV 12 Rewrite.md

```diff
@@ -0,0 +1,56 @@
+# Interpretation Exercise 01
+
+---
+
+## Proposed Rewrite of GOV 12
+
+To redress the aforementioned concerns, I would rewrite GOV 12 to achieve three aims. 
+
+First, I create a general protocol for GOV 12 and restrict its application to "substantial threats" to the functioning of Maker Governance. This is a far higher showing to meet than the current one. It redresses the risk stemming from ambiguity/overbreadth, while preserving to a degree the original "bootstrapping" intention.
+
+Second, I require endorsements from rotating ecosystem actors to provide checks/balances against the concentration of power in Governance Facilitators.
```

Comment:
Good, I think this is a sensible requirement.

Reply:
One possible thing is that you shouldn't be asking advisory council members directly, and that it should go through the facilitator. Hard to say though, might be fine.